third-capstone/README.md

```diff
@@ -1,8 +1,127 @@
-# Getting Started with Create React App
+<div align="center">
+  <h1><b>Movie character profile</b></h1>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [Tech Stack](#tech-stack)
+  - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Getting Started with Create React App](#getting-started-React)
+  - [Available Scripts](#available-scripts)
+  - [Learn More](#learn-more)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Movie character profile <a name="about-project"></a>
+
+**Movie character profile**  is a website for showing movie character profiles. It is a Single Page App (SPA) that allows users to:
+- View actors profile simple.
+to build this application, I use React,Redux.
+
+# 👁‍🗨 Preview
+
+Pleaseholder
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Languages</summary>
+  <ul>
+    <li><a href="https://html.spec.whatwg.org/">HTML5 / CSS / JavaScript</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>IDE</summary>
+  <ul>
+    <li><a href="https://code.visualstudio.com/">Visual Studio Code</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Hosting</summary>
+  <ul>
+    <li><a href="https://github.com/">GitHub</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Library</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Make simple calculations**
+- **Read a random math-related quote**
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+- [Live Demo Link](https://atatm.github.io/thirs-capstone/)
+
```

Comment:
- [ ] A valid link to the deployed project should be present here. Let's add it.